Core/include/Acts/Vertexing/HelicalTrackLinearizer.ipp

```diff
@@ -79,12 +79,9 @@ Acts::Result<Acts::LinearizedTrack> Acts::
   // q over p
   ActsScalar qOvP = paramsAtPCA(BoundIndices::eBoundQOverP);
 
-  // Get mass hypothesis from propagator options
-  ActsScalar m0 = pOptions.mass;
-  // Assume unit charge
-  // TODO: Get charge hypothesis from propagator options once they are included
-  // there.
-  ActsScalar p = std::abs(1. / qOvP);
+  ActsScalar m0 = params.particleHypothesis().mass();
+  ActsScalar p = std::abs(params.particleHypothesis().absoluteCharge() / qOvP);
```

Comment:
@felix-russo Just to inform you that this in principle can change behaviour, but should resolve the `TODO` that was there correctly

Reply:
Looks great! 👍